README.md

```diff
@@ -0,0 +1,50 @@
+# koa-esm-transform
+
+Middleware for Koa servers that transforms standard JavaScript modules to earlier versions of JavaScript and/or AMD modules (inlining loader script `@polymer/esm-amd-loader` into the HTML), for use with older browsers.  
+
+Consider an HTML file containing the following inline JavaScript module:
+
+```html
+<script type="module">
+import {someFunction} from './some-module.js';
+someFunction();
+</script>
+```
+
+The koa-esm-to-amd middleware would transform that HTML code to something like the following:
+
+```html
+<script>
+// An inline loader script from `@polymer/esm-amd-loader` package
+// which adds the `define` function used below.
+</script>
+<script>
+define(["./some-module.js"], function (_someModule) {
+  "use strict";
+
+  (0, _someModule.someFunction)();
```

Comment:
Since it's just an example, I'd remove the odd `(0, ...)()` thing that Babel is doing. Also maybe remove the `use strict` and s/_someModule/someModule.

Reply:
Removed! Thanks. It does read better now.